.eslintignore

```diff
@@ -4,3 +4,5 @@ jest.e2e.config.js
 license-header.js
 rollup.config.js
 tsconfig.json
+# The only check that applies here is really the license header:
+.eslintrc.js
```

Comment:
As this file was a `.js` file, we weren't actually passing it through eslint anyway, however, it still complained about the license header missing, hence, exclude the configuration file from eslint itself.